taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -127,27 +140,49 @@ def gather_files(self):
                 destination path and the checksum of the source file 
                 (or None if source is a folder)
         """
+        def _get_hash(sourcepath,destpath):
+            hash = None
```

Comment:
hmm did you know that `hash` is actually built-in method in Python? Maybe you could use it, and in any case you're overwriting it here, just FYI :)

```
In [1]: hash?
Type:        builtin_function_or_method
String form: <built-in function hash>
Namespace:   Python builtin
Docstring:
hash(object) -> integer

Return a hash value for the object.  Two objects with the same value have
the same hash value.  The reverse is not necessarily true, but likely.
```